admob/addon_template/AdmobExportPlugin.gd

```diff
@@ -59,7 +59,8 @@ class AndroidExportPlugin extends EditorExportPlugin:
 
 
 	func _get_android_manifest_application_element_contents(platform: EditorExportPlatform, debug: bool) -> String:
-		var __admob_node: Admob = _get_admob_node(EditorInterface.get_edited_scene_root())
+		var main_scene = load(ProjectSettings.get_setting("application/run/main_scene")).instantiate()
```

Comment:
@syntaxerror247, what do you think about checking the main scene if there is no Admob node in the "edited scene"?  That way we would (1) be "backward compatible" in terms of habits of existing user base & (2) search both the edited and main scenes before throwing an error.

Reply:
Yes I think it would be good

Reply:
@cengiz-pz I am going to move loading of main_scene inside condition block before checking in main_scene. Okay ?

Reply:
Yes, that would be great.  And printing an error message if node is not found on either scene would be a great way to let the users know that something failed.

Reply:
Ok just give me few minutes

Reply:
@cengiz-pz  what should be the error message